cmd/validate/image_test.go

```diff
@@ -759,3 +802,57 @@ func Test_WarningOutput(t *testing.T) {
 		}
 	  }`, mockPublicKey, mockPublicKey), out.String())
 }
+
+func setupFulcioRoots(t *testing.T) {
+	// Do not use afero.NewMemMapFs() here because the file is read by cosign
+	// which does not understand the filesystem-from-context pattern
+	f, err := os.Create(path.Join(t.TempDir(), "fulcio.pem"))
+	assert.NoError(t, err)
+	defer f.Close()
+	_, err = f.Write([]byte(hd.Doc(`
```

Comment:
Could use [embed](https://pkg.go.dev/embed) here, then these files could be descriptive by file name